booking-app/src/main/java/org/motechproject/bookingapp/domain/VisitBookingDetails.java

```diff
@@ -43,6 +45,36 @@ public VisitBookingDetails(Visit visit, LocalDate bookingActualDate) {
         this.bookingActualDate = bookingActualDate;
     }
 
+    public VisitBookingDetailsDto toDto() {
+
+        Subject participant = getVisit().getSubject();
+
+        VisitBookingDetailsDto dto = new VisitBookingDetailsDto();
+        dto.setBookingId(getId());
+        dto.setParticipantId(participant.getSubjectId());
+        dto.setParticipantName(participant.getName() + " " + participant.getHouseholdName());
+        dto.setFemaleChildBearingAge(isFemaleChildBearingAge(participant));
+
+        for (Visit participantVisit : participant.getVisits()) {
+            if (participantVisit.getType() == VisitType.SCREENING && participantVisit.getDate() != null) {
+                dto.setActualScreeningDate(participantVisit.getDate());
+            }
+        }
+
+        dto.setDate(getBookingPlannedDate());
+        dto.setStartTime(getStartTime());
+        dto.setEndTime(getEndTime());
+        dto.setClinicId(getClinic().getId());
+        dto.setSiteId(getClinic().getSite().getId());
+        dto.setVisitId(getVisit().getId());
+        dto.setLocation(getClinic().getSite().getSiteId() + " - " + getClinic().getLocation());
+        return dto;
+    }
+
+    private boolean isFemaleChildBearingAge(Subject participant) {
```

Comment:
Seth requested that this should not be calculated, but user should choose on the UI the value